example/t3dn_bip_example/previews.py

```diff
@@ -3,17 +3,26 @@
 
 folder = None
 collection = None
+PREVIEW_COLL = {}
```

Comment:
I would not use a dictionary for this example. Just have two variables instead. its too complicated for an simple example

Reply:
I've changed this to a single variable, for the hero image. I'm using the already existing `collection` variable to store the enum icons.